sphinx/ext/autosummary/generate.py

```diff
@@ -104,12 +104,38 @@ def _underline(title, line='='):
 
 # -- Generating output ---------------------------------------------------------
 
+def _should_skip_inherited_member(obj, membername, inherited_members):
+    """Determine if a member should be omitted from documentation according to
+     the inherited_members config value.
+    """
+
+    assert inherited_members in ('all', 'overridden', 'none'), inherited_members
+    if inherited_members == 'all':
+        # Document all inherited members
+        return False
+
+    method = getattr(obj, membername)
+    base_methods = [
+        getattr(base, membername) for base in obj.__bases__ if hasattr(base, membername)]
+
+    if inherited_members == 'overridden':
+        # Document only overridden members
+        return any(_ is method for _ in base_methods)
+    elif inherited_members == 'none':
+        # Document only new members
+        return len(base_methods) > 0
+    else:
+        assert False
+
+
 def generate_autosummary_docs(sources, output_dir=None, suffix='.rst',
                               warn=_simple_warn, info=_simple_info,
                               base_path=None, builder=None, template_dir=None,
-                              imported_members=False):
+                              imported_members=False, inherited_members=None):
     # type: (List[unicode], unicode, unicode, Callable, Callable, unicode, Builder, unicode, bool) -> None  # NOQA
```

Comment:
This line is the cause of your Travis CI build failure. I imagine you need to add `unicode` to the first tuple, like so:

    # type: (List[unicode], unicode, unicode, Callable, Callable, unicode, Builder, unicode, bool, unicode)